app/Models/Permission.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsToMany;
+
+class Permission extends Model
+{
+    use HasFactory;
+
+    protected $fillable = ['name'];
```

Comment:
We don't need mass assignment protection so you can remove the fillable properties from the models and add `Model::unguard()` in the `AppServiceProvider` `boot` method.

Whenever you use the `create()` or the `update()` method on an Eloquent model, you will have to specify all values that will be created or updated.